.github/workflows/main.yml

```diff
@@ -34,7 +34,7 @@ jobs:
         fetch-depth: 0
 
     - name: Add msbuild to PATH
-      uses: microsoft/setup-msbuild@v1.0.1
+      uses: microsoft/setup-msbuild@v1
```

Comment:
Looks good.  We need to make sure to use the shortest version, "v#", possible to allow it to automatically pick the latest version.